content/sensu-go/6.1/observability-pipeline/observe-schedule/subscriptions.md

```diff
@@ -0,0 +1,158 @@
+This makes it possible to generate ad hoc check requests that target specific entities via the API.
+
+{{% notice note %}}
+**NOTE**: You can directly add, update, and delete subscriptions for individual entities via the backend with [sensuctl](../../../sensuctl/create-manage-resources/#update-resources), the [entities API](../../../api/entities/), and the [web UI](../../../web-ui/view-manage-resources/#manage-entities).
+{{% /notice %}}
+
+## Publish checks
+
+If you want Sensu to automatically schedule and execute a check according to its subscriptions, set the [`publish` attribute][12] to `true` in the check definition.
+
+You can also manually schedule [ad hoc check execution][11] with the [check API][16], whether the `publish` attribute is set to `true` or `false`.
+To target the subscriptions defined in the check, include only the check name in the request body (e.g. `"check": "check_cpu"`).
+To override the check's subscriptions and target an alternate entity or group of entities, add the subscriptions attribute to the request body:
+
+{{< code shell >}}
+{
+  "check": "check_cpu",
+  "subscriptions": [
+    "entity:i-424242",
+    "entity:i-828282"
+  ]
+}
+{{< /code >}}
+
+## Example
+
+Suppose you have a Sensu agent with the `linux` subscription:
+
+{{< code shell >}}
+sensu-agent start --subscriptions linux --log-level debug
+{{< /code >}}
+
+For this agent to run a check, you must have at least one check with `linux` specified in the `subscriptions` attribute, such as this check to collect status information:
+
+{{< language-toggle >}}
+
+{{< code yml >}}
+type: CheckConfig
+api_version: core/v2
+metadata:
+  name: collect_info
+  namespace: default
+spec:
+  command: collect.sh
+  handlers:
+  - slack
+  interval: 10
+  publish: true
+  subscriptions:
+  - linux
+{{< /code >}}
+
+{{< code json >}}
+{
+  "type": "CheckConfig",
+  "api_version": "core/v2",
+  "metadata": {
+    "namespace": "default",
+    "name": "collect_info"
+  },
+  "spec": {
+    "command": "collect.sh",
+    "handlers": [
+      "slack"
+    ],
+    "interval": 10,
+    "publish": true,
+    "subscriptions": [
+      "linux"
+    ]
+  }
+}
+{{< /code >}}
+
+{{< /language-toggle >}}
+
+If this is your only check for the `linux` subscription, this is the only check that your agent will execute.
+If you add more checks that specify the `linux` subscription, your agent will automatically run those checks too (as long as the `publish` attribute is set to `true` in the check definitions).
+
+You can also add more subscriptions for your agent.
+For example, if you want your agent to execute checks for the `webserver` subscription, you can add it with the `subscriptions` flag:
+
+{{< code shell >}}
+sensu-agent start --subscriptions linux,webserver --log-level debug
+{{< /code >}}
+
+Now your agent will execute checks with the `linux` or `webserver` subscriptions.
```

Comment:
Suggestion: mention again and link to other methods to add/remove subscriptions to entities.